src/midiEditorCommands.cpp

```diff
@@ -729,6 +729,23 @@ void postMidiSelectNotes(int command) {
 	outputMessage(s);
 }
 
+void postMidiSelectEvents(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int evtIndex=-1;
+	int count=0;
+	for(;;){
+		evtIndex = MIDI_EnumSelEvts(take, evtIndex);
+		if (evtIndex == -1)
+			break;
+		++count;
+	}
+	// fakeFocus = FOCUS_NOTE;
+	ostringstream s;
+	s << count << " event" << ((count == 1) ? "" : "s") << " selected";
```

Comment:
This needs to be translatable. A lot of the MIDI editor code isn't translatable yet, but I'm not landing any new strings that aren't translatable. Search for "{} track copied" in reaper_osara.cpp for an example of how to do this, including the translators comment.